src/libs/PolicyUtils.ts

```diff
@@ -1205,6 +1206,50 @@ function canModifyPlan(policyID?: string) {
     return !!policy && isPolicyAdmin(policy);
 }
 
+function getAdminsPrivateEmailDomains(policyID?: string) {
```

Comment:
This is another method that breaks the reactivity of the view component via the use of `getPolicy()`. The view needs to connect to the policy collection directly using `useOnyx()` and then pass the full policy object to this method.

Reply:
great catch! thanks!